avocado/utils/network/interfaces.py

```diff
@@ -314,3 +314,29 @@ def remove_ipaddr(self, ipaddr, netmask):
         except Exception as ex:
             msg = 'Failed to remove ipaddr. {}'.format(ex)
             raise NWException(msg)
+
+    def move_interface_file(self, ignore_missing=True):
+        """
+        Revert interface file from backup
+
+        This method checks if a backup version  is available for given interface
+        then it copies backup file to interface file in /sysfs path
+        """
+
+        current_distro = distro_detect()
+
+        filename = "ifcfg-{}".format(self.name)
+        if current_distro.name in ['rhel', 'fedora']:
+            path = "/etc/sysconfig/network-scripts"
+        elif current_distro.name == 'SuSE':
+            path = "/etc/sysconfig/network"
+        else:
+            msg = 'Distro not supported by API. Could not save ipaddr.'
+            raise NWException(msg)
+
+        backup_file = "{}.backup-{}".format(filename, time.time())
+        if os.path.exists(backup_file):
+            shutil.move(backup_file, filename)
```

Comment:
This logic makes no sense to me: You are moving a backup_file that will never exist (because of the time.time()).

Also during a few PRs ago, we removed the `time.time()` logic in order to avoid having a dir full of files.

Reply:
ah sure i miss that